Live-Snap/Live-Snap/Networking/SnapcodeEndpointRequest.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SnapcodeEndpointRequest.swift
+//  Live-Snap
+//
+//  Created by Oleg Abalonski on 1/6/18.
+//  Copyright © 2018 Oleg Abalonski. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+
+class SnapcodeEndpointRequest {
+    
+    var snapchatUsername: String
+    
+    init(snapchatUsername: String) {
+        self.snapchatUsername = snapchatUsername
+    }
+    
+    //https://github.com/Alamofire/AlamofireImage
+    //Shouldn't we be Almofire.download the image???
```

Comment:
Multiple ways to download an image from a url, doesn't matter which one we use since it'll accomplish the same task